revocation/api.go

```diff
@@ -208,8 +192,8 @@ func (acc *Accumulator) Sign(sk *PrivateKey) (*SignedAccumulator, error) {
 }
 
 // Remove generates a new accumulator with the specified e removed from it.
-func (acc *Accumulator) Remove(sk *PrivateKey, e *big.Int, parent *Event) (*Accumulator, *Event, error) {
-	eInverse, ok := common.ModInverse(e, new(big.Int).Mul(sk.P, sk.Q))
+func (acc *Accumulator) Remove(sk *keys.PrivateKey, e *big.Int, parent *Event) (*Accumulator, *Event, error) {
+	eInverse, ok := common.ModInverse(e, new(big.Int).Mul(sk.PPrime, sk.QPrime))
```

Comment:
In fact, it is a pure refactor. This is because the `revocation` subpackage uses and has always used only `PPrime` and `QPrime` (or rather their product, i.e. `gabikeys.PrivateKey.Order`). On `master`, you can see them entering `revocation` [here](https://github.com/privacybydesign/gabi/blob/b4e094b964d8515e49305c6f2add47f4bed3b192/keys.go#L170). There they are assigned to `revocation.PrivateKey.P` and `revocation.PrivateKey.Q` fields, which were dereferenced here.

Since `revocation` actually only needs the `Order` member of `gabikeys.PrivateKey`, I have now refactored it such that it only uses that member and never `PPrime` or `QPrime` anymore.